FordLabs/PeopleMover
# 183 remove old permissions experience

## Issue
Resolves 183

## What was done
- [x] Removed old UI experience for share access form
- [x] Removed supporting infrastructure + tests for share access form

## How to test
1. Enter a space
2. Open the share access form
3. Verify that you can change owners, invite editors, remove own access + get redirected
